sql/sql_parse.cc

```diff
@@ -573,13 +573,10 @@ void init_sql_command_flags() {
   server_command_flags[COM_STMT_RESET] |= CF_SKIP_WSREP_CHECK;
   server_command_flags[COM_STMT_SEND_LONG_DATA] |= CF_SKIP_WSREP_CHECK;
   server_command_flags[COM_QUIT] |= CF_SKIP_WSREP_CHECK;
-  server_command_flags[COM_PROCESS_INFO] |= CF_SKIP_WSREP_CHECK;
-  server_command_flags[COM_PROCESS_KILL] |= CF_SKIP_WSREP_CHECK;
```

Comment:
Why this change? Is it because upstream removed `COM_PROCESS_INFO`, `COM_PROCESS_KILL` and `COM_PROCESS_KILL`